src/0_queue.js

```diff
@@ -15,6 +15,6 @@ var Queue = function() {
 	};
 	return function(task) {
 		queue.push(task);
-		if (queue.length == 1) { next(); }
```

Comment:
I don't think this is right solution, but it got it working properly for now.

It looks like the queue never clears, so this condition would stop the close() from running